network-api/networkapi/templates/partials/banner.html

```diff
@@ -0,0 +1,47 @@
+{% load settings_value i18n static %}
+
+{% with wrapper_class="tw-py-7" %}
+<div class="donate-banner tw-bg-black dark-theme {{wrapper_class}} print:tw-hidden tw-w-full">
+    <a href="#" class="p-4 tw-box-content banner-close" aria-label="Close Banner">
```

Comment:
This needs to be translated as well ;)

```suggestion
    <a href="#" class="p-4 tw-box-content banner-close" aria-label="{% trans "Close Banner" %}">
```